csfieldguide/templates/interactives/prototype-whole-page-interactive-base.html

```diff
@@ -0,0 +1,31 @@
+{% load static %}
```

Comment:
Should `csfieldguide/templates/interactives/prototype-whole-page-interactive-base.html` still be included?